src/nunit-gui/nunit-gui.csproj

```diff
@@ -34,15 +34,15 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Mono.Cecil, Version=0.9.6.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\NUnit.Engine.3.5.0\lib\Mono.Cecil.dll</HintPath>
+      <HintPath>..\..\packages\NUnit.Engine.3.6.0-ci-03382-pr-146\lib\Mono.Cecil.dll</HintPath>
```

Comment:
@CharliePoole is there a way in nuget settings to make these paths static? if not no worries.